common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -161,7 +161,7 @@ trait FeatureSwitches {
     "If this is switched on then videos are enhanced using VideoJS",
     owners = Seq(Owner.withGithub("siadcock")),
     safeState = On,
-    sellByDate = new LocalDate(2018, 8, 31),
+    sellByDate = new LocalDate(2018, 9, 30),
```

Comment:
teamcity seems to be saying this has to be a weekday

Reply:
🙄